### PR TITLE
Refactor context menu UI

### PR DESCRIPTION
**Pull request by 08A**
## About the PR
The PR https://github.com/space-wizards/space-station-14/pull/14308 seems to be a controversial proposition. So I am splitting the refactoring work and feature.

**Media**

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

### CODE REVIEW

File: Content.Client/ContextMenu/UI/EntityMenuElement.cs [hunk 1]
```diff
@@ -31,33 +24,30 @@ public sealed partial class EntityMenuElement : ContextMenuElement
...
+            get => _count;
+            private set {
+                _count = value;
+                if (_count > 1)
+                {
+                    IconLabel.Text = _count.ToString();
+                    IconLabel.Visible = true;
+                }
+            }
+        }
```
Comment (moonheart08) [inline Content.Client/ContextMenu/UI/EntityMenuElement.cs:38]: Only ever set in UpdateCount, does not need encapsulated.

Comment (08A) [reply Content.Client/ContextMenu/UI/EntityMenuElement.cs:38]: Count is also set in the constructor and `Dispose` method. If I remove the setter I will have to change the default visibility of the `IconLabel`. Is this an acceptable solution for you ?

Comment (08A) [reply Content.Client/ContextMenu/UI/EntityMenuElement.cs]: I removed the custom setter. What do you think ?

**Review by AJCM-git — APPROVED** [review]
havent tested in-game but the code looks aight

**Comment by 08A** [comment]
Hi @moonheart08 and @AJCM-git , sorry to ping it has been a week since your last review. Is this mergeable ?

**Comment by AJCM-git** [comment]
> Hi @moonheart08 and @AJCM-git , sorry to ping it has been a week since your last review. Is this mergeable ?

The entity menu no longer groups entities if the ones in range all share the same uid

**Comment by 08A** [comment]
> The entity menu no longer groups entities if the ones in range all share the same uid

Are you talking about a bug on this PR or a new feature on master ?

PS: I don't fully understand your comment. Do you have any example ?

**Comment by AJCM-git** [comment]
> Are you talking about a bug on this PR or a new feature on master ?
> 
> PS: I don't fully understand your comment. Do you have any example ?

Sorry, meant prototype, also nevermind, its a master bug

**Pull request merged**